daemon/algod/api/algod.oas2.json

```diff
@@ -1693,18 +1693,48 @@
           "type": "string"
         },
         "sig-type": {
-          "description": "Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig",
+          "description": "Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig\n* or null if unknown",
           "type": "string",
           "enum": [
             "sig",
             "msig",
             "lsig"
           ]
         },
+        "total-apps-opted-in": {
+          "description": "The count of all application local data (AppLocalState objects) stored in this account.",
+          "type": "integer"
+        },
+        "total-assets-opted-in": {
+          "description": "The count of all assets (AssetHolding objects) held by this account.",
+          "type": "integer"
+        },
+        "total-created-apps": {
+          "description": "The count of all apps (AppParams objects) created by this account.",
+          "type": "integer"
+        },
+        "total-created-assets": {
+          "description": "The count of all assets (AssetParams objects) created by this account.",
+          "type": "integer"
+        },
         "auth-addr": {
           "description": "\\[spend\\] the address against which signing should be checked. If empty, the address of the current account is used. This field can be updated in any transaction by setting the RekeyTo field.",
           "type": "string",
           "x-algorand-format": "Address"
+        },
+        "deleted": {
```

Comment:
IMO this only makes sense for the indexer — algod doesn't track historical accounts.